### PR TITLE
Integrate suse rocky image for nova

### PR DESCRIPTION
**Pull request by rajdharwadkar**

### CODE REVIEW

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 1]
```diff
@@ -80,15 +80,36 @@ data:
     conf:
       ceph:
         enabled: true
-        monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
+        #monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
```
Comment (toabctl) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: Why do we need to comment this?

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 2]
```diff
@@ -80,15 +80,36 @@ data:
...
+        #admin_keyring: {{ ceph_admin_keyring_key }}
```
Comment (toabctl) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: same question here

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 3]
```diff
@@ -80,15 +80,36 @@ data:
...
         admin_keyring: {{ ceph_admin_keyring_b64key | b64decode }}
         cinder:
-          user: {{ ses_cluster_configuration.cinder.rbd_store_user }}
-          keyring: {{ ses_cluster_configuration.cinder.key }}
+          user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
```
Comment (toabctl) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: How is that change related to the commit msg which is about the nova rocky image?

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml (hunk 1 above)
Comment (jgu17) [reply site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: I think we need the monitors, @rajdharwadkar ?

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 4]
```diff
@@ -80,15 +80,36 @@ data:
...
+          keyring: {{ suse_ses_cinder_pool_key | default( ceph_admin_keyring_b64key | b64decode) }}
```
Comment (jgu17) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: can you explain why we are changing to use suse_ses_cinder_pool_key which seems to be deploy_osh var?

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 5]
```diff
@@ -80,15 +80,36 @@ data:
...
+          AllowOverride All
+          <IfModule !mod_access_compat.c>
+            Require all granted
+          </IfModule>
+          <IfModule mod_access_compat.c>
+            Order allow,deny
+            Allow from all
+          </IfModule>
+        </Directory>
+      #nova:
```
Comment (jgu17) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: so what is the libvirt type if not spec'ed here?

File: site/soc/software/config/versions.yaml [hunk 6]
```diff
@@ -41,6 +41,11 @@ data:
         reference: 8f290d3b8d2a8d787f107d05b3be0199d33d7026
         subpath: helm-toolkit
         type: git
+      nova-htk:
+        location: https://git.openstack.org/openstack/openstack-helm-infra
+        reference: 8f290d3b8d2a8d787f107d05b3be0199d33d7026
```
Comment (jgu17) [inline site/soc/software/config/versions.yaml]: please update this commit id here to 5e1ecd9840397bf9e8829ce0d98fcb721db1b74e the same as the value from PR 294. The upstream has uplifted the htk version used in Airship.

File: site/soc/software/charts/osh/openstack-compute-kit/nova.yaml [hunk 7]
```diff
@@ -83,9 +83,29 @@ data:
         monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
         admin_keyring: {{ ceph_admin_keyring_b64key | b64decode }}
         cinder:
-          user: {{ ses_cluster_configuration.cinder.rbd_store_user }}
+          user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
```
Comment (jgu17) [inline site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: was the old style broken or new style is preferred?

Comment (rajdharwadkar) [reply site/soc/software/charts/osh/openstack-compute-kit/nova.yaml]: I saw that osh team is using that style and thought it should be consistent with that code.  But I see many places to be fixed in our code. so will keep our style.
